diff.py

```diff
@@ -1835,21 +1932,11 @@ def process(dump: str, config: Config) -> List[Line]:
         # transforming 'row' into a coarser version that ignores registers and
         # immediates.
         original = row
-
-        while i < len(lines):
+        if i < len(lines):
```

Comment:
this does need to be a while -- there can be multiple relocations that apply to the same line. In practice the case I know of is a silly one where all but one relocation are "no-op" relocations, so the duplicate processing is harmless

Reply:
Done, reverted.